phpBB/adm/style/acp_posting_buttons.html

```diff
@@ -1,5 +1,7 @@
 <script type="text/javascript">
 // <![CDATA[
+	var mention_url = '{UA_MENTION_URL}';
+	var mention_topic_id = '{S_TOPIC_ID}';
```

Comment:
`S_` means `switch`, i.e. boolean value. Just use `{TOPIC_ID}` which is maybe already defined.

Reply:
@senky `S_TOPIC_ID` is indeed already defined in `functions.php`.
That is why it is used here.

Reply:
According to guidelines, `S_` stands for `system`, not for `switch`.

Reply:
Ah right, then it's OK.